### PR TITLE
Add Xiaomi 4 and 4 Pro Documentation

### PR DESCRIPTION
**Pull request by borky**
## Proposed change

Add Xiaomi 4 and 4 Pro Documentation

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: [75745](https://github.com/home-assistant/core/pull/75745)
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/xiaomi_miio.markdown [hunk 1]
```diff
@@ -216,6 +216,8 @@ Supported devices:
 | Air Purifier 3 (2019)  | zhimi.airpurifier.ma4  | |
 | Air Purifier 3H (2019) | zhimi.airpurifier.mb3  | |
 | Air Purifier 3C        | zhimi.airpurifier.mb4  | |
+| Air Purifier 4         | zhimi.airp.mb5         | |
+| Air Purifier 4 PRO     | zhimi.airp.vb4         | |
```
Comment (bieniu) [inline source/_integrations/xiaomi_miio.markdown:220]: Why are you using `zhimi.airp.mb5` here instead of `zhimi.airpurifier.mb5`?

Comment (borky) [reply source/_integrations/xiaomi_miio.markdown:220]: that is the identifier for these 2 models and it is according to the python-miio library(https://github.com/rytilahti/python-miio/blob/3c6b8dca6bc799e2d8df8f506e26fae3d5f424e4/miio/integrations/airpurifier/zhimi/airpurifier_miot.py#L245)

Comment (bieniu) [reply source/_integrations/xiaomi_miio.markdown:220]: If so, I think those model names should also be used on line 501.

Comment (borky) [reply source/_integrations/xiaomi_miio.markdown:220]: thanks, i updated it.

**Review by bieniu — APPROVED** [review]

**Review by frenck — APPROVED** [review]

**Pull request merged**